uiowa/uiowa
# Adds `Padding: all sides minimal` style for text area and webform blocks

Resolves https://github.com/uiowa/uiowa/issues/6905

In this pr:
- Adds `Padding: all sides minimal` style
  - Would `Padding: all sides compact` be better?
- Changes the padding group from multiselect to single select. 

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /test-sidebar
```
- Code review
- Test the webform block and the text area block on this page and confirm that the `Padding: all sides minimal` style is available and sets the block padding to `2rem`. 
- Test other pages and webform blocks to see if they are or are not affected. 
- Test with gray background plus minimal padding setting.
